src/OrbitQt/ProfilingTargetDialog.cpp

```diff
@@ -462,10 +463,15 @@ void ProfilingTargetDialog::OnProcessListUpdate(
     // TrySelectProcess attempts to select the process again if it exists
     // in process_model_.
     if (process_ != nullptr) {
-      bool success = TrySelectProcess(*process_);
+      bool success = TrySelectProcess(process_->name());
```

Comment:
What happens when two processes have the same name? (Maybe a game forks or whatever). Is there a chance, that the selection jumps around between these two (or more) processes with the same name?

Reply:
Good point. Yes, that could happen, if the processes switch positions due to changing CPU load. If you specify a `process_name` parameter, the Dialog will automatically close after finding a matching process, so you won't see any jumping of selections, but will end up with the first of multiple matches selected.

The jumping will occur when you click "End Session" and go back to the startup window. But this problem isn't newly introduced, the implementation before was only checking the process name in `TrySelectProcess` as well...

I'm not sure if this is something we should take care of. We could pass an optional PID to `TrySelectProcess` and pick that if there's multiple options, WDYT? But I'd probably create another PR for this.

Reply:
Ahh, I'm sorry. I didn't realize that the previous implementation had the same problem. So I'm fine with shipping it like this.

Of course, matching a PID would be better, but I don't want to put the burden of fixing random discovered bugs on you.